### PR TITLE
New CLI init command

### PR DESCRIPTION
**Pull request by Jolg42**
Waiting for default schema and url

### CODE REVIEW

File: cli/introspection/src/commands/Init.ts [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
 
+export const defaultSchema = `// This is your Prisma schema file,
+// Learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// The \`datasource\` block is used to specify the connection to your DB.
+// Set the \`provider\` field to match your DB type: "postgresql", "mysql" or "sqlite".
+// The \`url\` field must contain the connection string to your DB.
+// Learn more about connection strings for your DB: https://pris.ly/connection-strings
+datasource db {
+provider = "postgresql" // other options are: "mysql" and "sqlite"
```
Comment (janpio) [inline cli/introspection/src/commands/Init.ts]: should probably be indented

Comment (Jolg42) [reply cli/introspection/src/commands/Init.ts]: Yes, waiting for final version to fix :)

**Comment by Jolg42** [comment]
Only one url to update now :)

**Comment by Jolg42** [comment]
It is ready!

**Pull request merged**